CMake/lrs_options.cmake

```diff
@@ -4,6 +4,7 @@ option(BUILD_GLSL_EXTENSIONS "Build GLSL extensions API" ON)
 option(BUILD_WITH_OPENMP "Use OpenMP" OFF)
 option(BUILD_WITH_TM2 "Build with support for Intel TM2 tracking device" ON)
 option(BUILD_EASYLOGGINGPP "Build EasyLogging++ as a part of the build" ON)
+option(BUILD_AUS "Build AUS a part of the build" ON)
```

Comment:
I suggest calling it "Enable AUS"
Than the comment could be "Enable collectimg and publishing AUS"